index.html

```diff
@@ -15,43 +15,46 @@
+          </div>
+          <nav>
+            <ul class="menu">
+              <li><a href="#portfolio">Portfolio</a></li>
+              <li><a href="#about">About</a></li>
+              <li><a href="#contact">Contact</a></li>
+            </ul>
+          </nav>
+        </div>
+      </div>
     </header>
     <main>
-    <section class="headline">
-          <h1 class="primary-text-2">I'm Zermarden <br>
-          Glad to see <br>
-          you!</h1>
-          <p class="primary-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-          <div class="social-media">
-            <div class="supporting-text">
-              <span>LET'S CONNECT!</span>
-            </div>
-          </div>
-          <div class="social-media-icons">
-            <ul class="social-media-icons-list">
-              <li><a href="#"><img src="assets/twitter.svg" alt="Twitter" width="20" height="20"></a></li>
-              <li><a href="#"><img src="assets/linkedin.svg" alt="Linkedin" width="20" height="20"></a></li>
-              <li><a href="#"><img src="assets/m.svg" alt="Medium" width="20" height="16"></a></li>
-              <li><a href="#"><img src="assets/git.svg" alt="GitHub" width="18" height="20"></a></li>
-              <li><a href="#"><img src="assets/finger.svg" alt="Peace" width="15" height="20"></a></li>
-          </ul>
-          </div>
-        </section>
-        <!--Images should be in Headline section. After that check CSS-->
+      <section class="headline">
+        <h1 class="hello">I'm Zermarden<br>
+          Glad to see you!</h1>
+        <p class="hello-text">I’m a software developer! I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a project you need 
+            coded, don’t hesitate to contact me. </p>
+        <h4 class="let-s-connect">LET'S CONNECT!</h4>
+        <ul class="social-list">
+          <li><a href="#"><img src="assets/twitter.svg" alt="Twitter" width="20" height="20"></a></li>
+          <li><a href="#"><img src="assets/linkedin.svg" alt="Linkedin" width="20" height="20"></a></li>
+          <li><a href="#"><img src="assets/m.svg" alt="Medium" width="20" height="16"></a></li>
+          <li><a href="#"><img src="assets/git.svg" alt="GitHub" width="18" height="20"></a></li>
+          <li><a href="#"><img src="assets/finger.svg" alt="Peace" width="15" height="20"></a></li>
+        </ul>
         <img class="header-shapes" src="assets/bg.png" alt="header-shapes">
-        <img class="header-background" src="assets/header-background-mobile.svg" alt="background-image">
-        <div class="works"> <!--Change it to section-->
+      </section>
+      <section class="works" id="portfolio">
           <div class="card-works-1">
```

Comment:
- [ ] [OPTIONAL] The page background color is not the same on mobile and desktop, kindly fix this so the color is united across all the screen sizes

|Desktop view|Mobile view|
|-|-|
|![Screenshot_20220604_091543](https://user-images.githubusercontent.com/25227022/171991109-76f25540-a083-4d3e-91bb-c6199fbaaa0f.png)|![Screenshot_20220604_091527](https://user-images.githubusercontent.com/25227022/171991112-0eafb707-ca1d-4289-b5b8-13087a4d8667.png)|